app/pages/sections/order-items/catalogue-solutions/delete/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "backLinkText": "Go back",
+    "title": "Delete {{solutionName}} from {{orderId}}?",
+    "description": "This Catalogue Solution will be deleted from all the Service Recipients in your order along with any of its Additional Services.",
+    "orderDescriptionTitle": "Order description",
+    "noButton": {
+      "text": "Cancel delete"
+    },
+    "yesButton": {
+      "text": "Confirm delete"
+    }
+  }
+  
```

Comment:
I think there is some surplus white space at the end of the file here. There should just be a single trailing new line.

Reply:
resolved